### PR TITLE
Change rbenv install for apple silicon

### PR DESCRIPTION
**Pull request by DenisZackharov**

### CODE REVIEW

File: modules/ruby.bash [hunk 1]
```diff
@@ -1,25 +1,21 @@
...
-  rbenv install $ruby_version
+if ! command -v rbenv &> /dev/null
+then
+  echo 'Installing rbenv'
+  # shellcheck disable=SC2016
+  [ -f "$HOME/.zprofile" ] && echo 'eval "$(/opt/homebrew/bin/brew shellenv)"' >> ~/.zprofile
+  # shellcheck disable=SC2016
+  [ -f "$HOME/.zshrc" ] && echo 'eval "$(rbenv init -)"' >> ~/.zshrc
+  brew install rbenv
+  brew upgrade rbenv ruby-build
```
Comment (DmitryBarskov) [inline modules/ruby.bash:16]: Why do we need to update rbenv after we just install it?
```suggestion
  brew install rbenv ruby-build
```

File: modules/ruby.bash [hunk 2]
```diff
@@ -1,25 +1,21 @@
...
-ruby_version="$(rbenv install -l | grep -E "^\\s*([.0-9]+)$" | tail -1)"
-
-if test -z "$(rbenv versions --bare|grep $ruby_version)"; then
-  info_echo "Install Ruby $ruby_version"
-  rbenv install $ruby_version
+if ! command -v rbenv &> /dev/null
+then
+  echo 'Installing rbenv'
+  # shellcheck disable=SC2016
+  [ -f "$HOME/.zprofile" ] && echo 'eval "$(/opt/homebrew/bin/brew shellenv)"' >> ~/.zprofile
```
Comment (DmitryBarskov) [inline modules/ruby.bash:12]: What is this line? I don't think it is related to `ruby.bash` file

File: modules/ruby.bash [hunk 3]
```diff
@@ -1,25 +1,21 @@
...
+  # shellcheck disable=SC2016
+  [ -f "$HOME/.zshrc" ] && echo 'eval "$(rbenv init -)"' >> ~/.zshrc
+  brew install rbenv
+  brew upgrade rbenv ruby-build
+  eval "$(rbenv init -)"
+  /bin/bash -c "curl -fsSL https://github.com/rbenv/rbenv-installer/raw/main/bin/rbenv-doctor | bash"
```
Comment (DmitryBarskov) [inline modules/ruby.bash:18]: Why is it here? It should be OK.

File: modules/functions.bash [hunk 4]
```diff
@@ -1,6 +1,6 @@
 export HOMEBREW_CASK_OPTS="--appdir=/Applications"
```
Comment (DmitryBarskov) [inline modules/functions.bash:1]: Also, please change back permissions of this file.

**Review by DmitryBarskov — CHANGES_REQUESTED** [review]

File: modules/ruby.bash (hunk 3 above)
Comment (DenisZackharov) [reply modules/ruby.bash:18]: In rbenv [documentation](https://github.com/rbenv/rbenv) wrote "Verify that rbenv is properly set up using this rbenv-doctor script"

File: modules/functions.bash (hunk 4 above)
Comment (DenisZackharov) [reply modules/functions.bash:1]: You about RBENV_ROOT and NVM_DIR?

Comment (DmitryBarskov) [reply modules/functions.bash:1]: No, I'm talking about file permissions
image

File: modules/ruby.bash (hunk 2 above)
Comment (DenisZackharov) [reply modules/ruby.bash:12]: This sc disable only for single quotes, or here can be some errors?

File: modules/functions.bash (hunk 4 above)
Comment (DenisZackharov) [reply modules/functions.bash:1]: Please help me, how I can do this?

File: modules/ruby.bash (hunk 1 above)
Comment (DenisZackharov) [reply modules/ruby.bash:16]: My bad

File: modules/ruby.bash (hunk 2 above)
Comment (DmitryBarskov) [reply modules/ruby.bash:12]: @DenisZackharov  I talk about `/opt/homebrew/bin/brew shellenv`

File: modules/functions.bash (hunk 4 above)
Comment (DmitryBarskov) [reply modules/functions.bash:1]: `chmod 644 modules/functions.bash`

File: modules/ruby.bash (hunk 2 above)
Comment (DenisZackharov) [reply modules/ruby.bash:12]: Do you mean that this line need move in brew.bash file?

File: modules/functions.bash (hunk 4 above)
Comment (DenisZackharov) [reply modules/functions.bash:1]: Thank you, but I don't remember that I chmod for this file, hmm

File: modules/ruby.bash [hunk 5]
```diff
@@ -1,25 +1,18 @@
...
-info_echo "Set default gems list"
-echo "bundler" >> "$(brew --prefix rbenv)/default-gems"
-
-info_echo "Installing latest stable Ruby..."
-ruby_version="$(rbenv install -l | grep -E "^\\s*([.0-9]+)$" | tail -1)"
-
-if test -z "$(rbenv versions --bare|grep $ruby_version)"; then
-  info_echo "Install Ruby $ruby_version"
-  rbenv install $ruby_version
+if ! command -v rbenv &> /dev/null
```
Comment (DmitryBarskov) [inline modules/ruby.bash:8]:
```suggestion
if  test ! "$(command -v rbenv)"
```

File: modules/ruby.bash [hunk 6]
```diff
@@ -1,25 +1,18 @@
...
+then
+  echo 'Installing rbenv'
```
Comment (DmitryBarskov) [inline modules/ruby.bash:10]: Let's use `info_echo` everywhere

**Review by DmitryBarskov — CHANGES_REQUESTED** [review]

**Review by DmitryBarskov — APPROVED** [review]

**Pull request merged**